src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -0,0 +1,105 @@
+<Meta title="SDK Providers/VoiceFocusProvider" />
+
+# VoiceFocusProvider
+
+The `VoiceFocusProvider` provides a function transforming a normal device to voice focus transform device, and the support state of voice focus status
+
+This provider is independent from `MeetingProvider`. You can put `VoiceFocusProvider` wherever you want in the component tree to control the workflow of voice focus.
+
+## State
+
+```javascript
+{
+  // Whether the voice focus is supported
+  isVoiceFocusSupported: boolean;
+
+  // A function to transform a device to voice focus transform device
```

Comment:
Should we explicitly say: "an audio input device to a voice focus audio transform device"?

Reply:
Use the term `an Amazon voice focus device` instead through out